### PR TITLE
Fix `log_query` to format SQL statement correctly in `AthenaOperator`

### LINKED ISSUE #34189: `log_query` of `AthenaOperator` dumps un readable query
### Body

Log:
Screenshot 2023-09-07 at 22 18 41

Compare to render tab:
Screenshot 2023-09-07 at 22 19 50

**Reproduce steps:**

set `file.sql` as:
```
CREATE EXTERNAL TABLE IF NOT EXISTS `outputs3`(
  `region` string,
  ....
)
```

set operator as:

```
AthenaOperator(
    task_id="my_task",
    query="file.sql",
    database="my_db",
    log_query=True,
    output_location="s3://...",
)
```
        
 **The needed fix:**
 Pretty print the query. It should appear exactly like it is in the `.sql` file.
 

### Committer

- [X] I acknowledge that I am a maintainer/committer of the Apache Airflow project.

### PR DESCRIPTION
**Pull request by ferruzzi**
Just some whitespace changes to make the Athena query log messages look more readable in the logs.

Before:
image

After:
image

closes: https://github.com/apache/airflow/issues/34189

### CODE REVIEW

File: airflow/jobs/scheduler_job_runner.py [hunk 1]
```diff
@@ -721,6 +721,7 @@ def _process_executor_events(self, session: Session) -> int:
             if state == TaskInstanceState.QUEUED:
                 ti.external_executor_id = info
                 self.log.info("Setting external_id for %s to %s", ti, info)
+                session.commit()
```
Comment (eladkal) [inline airflow/jobs/scheduler_job_runner.py]: I think this is unrelated change?

Comment (ferruzzi) [reply airflow/jobs/scheduler_job_runner.py]: Yeah, it's part of a different PR.  Thanks for checking.

File: airflow/providers/amazon/aws/executors/ecs/ecs_executor.py [hunk 2]
```diff
@@ -470,3 +482,44 @@ def get_container(self, container_list):
             if container["name"] == self.container_name:
                 return container
         raise KeyError(f"No such container found by container name: {self.container_name}")
+
+    def try_adopt_task_instances(self, tis: Sequence[TaskInstance]) -> Sequence[TaskInstance]:
```
Comment (syedahsn) [inline airflow/providers/amazon/aws/executors/ecs/ecs_executor.py]: I think the commits might have gotten mixed up here. This PR is doing 2 unrelated things it seems.

Comment (ferruzzi) [reply airflow/providers/amazon/aws/executors/ecs/ecs_executor.py]: Yeah, I had some git issues in another PR I was working on at the same time and thought I untangled them, but it looks like I didn't sort out this branch.

File: airflow/jobs/scheduler_job_runner.py (hunk 1 above)
Comment (ferruzzi) [reply airflow/jobs/scheduler_job_runner.py]: This is fixed now, sorry for the confusion.

File: airflow/providers/amazon/aws/executors/ecs/ecs_executor.py (hunk 2 above)
Comment (ferruzzi) [reply airflow/providers/amazon/aws/executors/ecs/ecs_executor.py]: This is fixed now, sorry for the confusion.

**Review by syedahsn — APPROVED** [review]
LGTM

**Review by dirrao — CHANGES_REQUESTED** [review]
Nice work. It would be great if you can add the test cases for this change.

**Review by Taragolis — APPROVED** [review]

**Review by hussein-awala — APPROVED** [review]

**Comment by ferruzzi** [comment]
@dirrao  I didn't add any test cases because it's just reformatting a string.  Could you suggest a test case you'd like to see?  I thought of two: one with a single-line query and one with a multi-line query which each assert the expected number of newlines in the log message, but that seems overly complicated for what it is... I think it would be something like "capture all logs, for each captured log message, if it contains 'Running Query with params' then assert it has the expected number of line breaks"...  It seems maybe excessive but I can add it if you think it's important.

**Comment by eladkal** [comment]
> Could you suggest a test case you'd like to see?

I think 1 unit test for `query_params_to_string` that gets a mock of `params` and verify the expected result of the function is enough.

**Comment by ferruzzi** [comment]
@eladkal  and @dirrao - Added two unit tests.  Let me know what you think.

**Review by o-nikolas — APPROVED** [review]

**Review by hussein-awala — APPROVED** [review]
LGTM

**Review by eladkal — APPROVED** [review]
